test/resources/activation_keys_test.rb

```diff
@@ -33,6 +33,10 @@ def test_create_and_update
 
       @name = "testakey1"
 
+      capture {
+        hammer.run(%W{activation-key delete --organization-label testcorp --name testakey1})
+      }
+
```

Comment:
No preference - I agree that having this at the end as cleanup makes more sense. Updated.